src/command_modules/azure-cli-redis/azure/cli/command_modules/redis/_help.py

```diff
@@ -22,32 +22,59 @@
     short-summary: Import data into a Redis cache.
 """
 
+helps['redis list'] = """
+    type: command
+    short-summary: Lists details about all caches within current Subscription or provided Resource Group.
+"""
+
 helps['redis import-method'] = """
     type: command
-    short-summary: Import data into a Redis cache.
+    short-summary: (DEPRECATED) Import data into Redis cache.
+    long-summary: |
+        WARNING: This command is deprecated. Instead, use the `import` command.
 """
 
-helps['redis list'] = """
+helps['redis update'] = """
+    type: command
+    short-summary: Scale or update settings of a Redis cache.
+"""
+
+helps['redis patch-schedule'] = """
+    type: group
+    short-summary: Manage Redis patch schedules.
+"""
+
+helps['redis patch-schedule create'] = """
     type: command
-    short-summary: List Redis caches.
+    short-summary: Create patching schedule for Redis cache.
 """
 
-helps['redis list-all'] = """
+helps['redis patch-schedule update'] = """
     type: command
-    short-summary: Gets all Redis caches in the specified subscription.
+    short-summary: Replace the patching schedule for Redis cache.
 """
 
-helps['redis update-settings'] = """
+helps['redis firewall-rules'] = """
+    type: group
+    short-summary: Manage Redis firewall rules.
+"""
+
+helps['redis firewall-rules create'] = """
     type: command
-    short-summary: Update the settings of a Redis cache.
+    short-summary: Create a redis cache firewall rule.
 """
 
-helps['redis update'] = """
+helps['redis firewall-rules update'] = """
     type: command
-    short-summary: Scale or update settings of a Redis cache.
+    short-summary: Update a redis cache firewall rule.
 """
 
-helps['redis patch-schedule'] = """
+helps['redis server-link'] = """
     type: group
-    short-summary: Manage Redis patch schedules.
+    short-summary: Manage Redis server links.
+"""
+
+helps['redis server-link create'] = """
```

Comment:
This command does not create a new resource but seems to set a link between two redis caches / or set a secondary redis cache.  If this is the case, will `az redis server-link set` make sense? I could see how it could go either way. 

Also it would be good to show an example of how to use this command.

Reply:
For this case, there are two patterns that are acceptable: create/delete (which is what it has) or add/remove.